base.js

```diff
@@ -296,7 +296,7 @@ module.exports = {
             {
                 patterns: [
                     {
-                        group: ['@salesforce/lds/*'],
+                        group: ['@salesforce/lds**'],
```

Comment:
AIUI this also captures `@salesforce/ldsfoobar`, which could be owned by some other team. (Unlikely, I admit, but possible.) I think this might work instead?
```suggestion
                        group: ['@salesforce/lds', '@salesforce/lds/*'],
```

Reply:
Note the `**`, I believe that covers deep imports like `@salesforce/lds/foo/bar/baz`?

Reply:
Ah yeah good catch 👍 Should be updated now. Let me know if you have any other concerns